packages/governance/src/validators.js

```diff
@@ -0,0 +1,65 @@
+// @ts-check
+
+import { E } from '@agoric/eventual-send';
+
+const { details: X, quote: q } = assert;
+
+/**
+ * Assert that the governed contract was started by the governor. Throws if
+ * either direction can't be established. If the call succeeds, then the
+ * governor got exclusive access to the governed contract's creatorFacet, and
+ * can be trusted to manage its parameters.
```

Comment:
How does this function establish anything about a `creatorFacet`? All I have to do to satisfy this check is to make one contract with a `.getGovernedContract()` and another with `electionManager` in its terms and point them at each other. I can post their `creatorFacet`s on the moon for all to access and still pass this check, no?